common/src/main/java/com/microsoft/identity/common/internal/providers/microsoft/azureactivedirectory/AzureActiveDirectory.java

```diff
@@ -72,6 +74,7 @@
     private static ConcurrentMap<String, AzureActiveDirectoryCloud> sAadClouds = new ConcurrentHashMap<>();
```

Comment:
We should set some parameters here - how many writers do we expect for this CHM?

Reply:
I'm not following. Kindly elaborate, what's CHM?

Reply:
ConcurrentHashMap.  By default, there are 16 shards here in the map, but if we only respect a single writer to this map, we don't need them.  the standard initialization parameters would be (16, 0.75f, 16) - We could take (16, 0.75f, 1) if we don't expect a lot of concurrent writes.